easybuild/tools/module_generator.py

```diff
@@ -226,6 +302,10 @@ class ModuleGeneratorTcl(ModuleGenerator):
     LOAD_REGEX = r"^\s*module\s+load\s+(\S+)"
     LOAD_TEMPLATE = "module load %(mod_name)s"
 
+    def __init__(self, *args, **kwargs):
+        """ModuleGeneratorTcl constructor."""
+        super(ModuleGeneratorTcl, self).__init__(*args, **kwargs)
```

Comment:
No, there is no reason, so better clean up the empty constructor in `ModuleGeneratorLua` instead (and drop this one here).